### PR TITLE
[qt] Fix misleading translation

### PR DESCRIPTION
**Pull request by maflcko**
The first commit fixes https://github.com/bitcoin/bitcoin/pull/7035#issuecomment-160808428

The second commit was on my list to fix before 0.12 but I somehow forgot it. People complain about this at transifex.

### CODE REVIEW

File: src/qt/locale/bitcoin_en.ts [hunk 1]
```diff
@@ -1153,7 +1153,7 @@
     </message>
     <message>
         <location line="+1"/>
-        <source>Reset all settings changes made over the GUI</source>
```
Comment (jonasschnelli) [inline src/qt/locale/bitcoin_en.ts]: Is the former (current) not more clear what `-resetguisettings` does?

Comment (paveljanik) [inline src/qt/locale/bitcoin_en.ts]: What about

Reset all settings changed in the GUI

Comment (jonasschnelli) [inline src/qt/locale/bitcoin_en.ts]: > Reset all settings changed in the GUI

Yes. This would be better IMO.

Comment (MarcoFalke) [inline src/qt/locale/bitcoin_en.ts]: Force pushed.

**Comment by jonasschnelli** [comment]
ACK fa5769e95a44427e1ed7d63795d4ff60985f0059 (no backport because of transiflex).

**Comment by MarcoFalke** [comment]
> no backport

No opinion on this but this could be merged into 0.12 without rebase. Is there an translation update before the final release?

**Comment by jonasschnelli** [comment]
> No opinion on this but this could be merged into 0.12 without rebase. Is there a translation update before the final release?

According to @laanwj, string freezing was done in 2015-12-01 (http://lists.linuxfoundation.org/pipermail/bitcoin-dev/2015-September/011182.html). People already translated 0.12er strings (https://www.transifex.com/bitcoin/bitcoin/qt-translation-012x/).

**Comment by laanwj** [comment]
Will pull translation *from* transifex a few times before the final 0.12 release, but it's too late to change strings.

File: src/torcontrol.cpp [hunk 2]
```diff
@@ -618,7 +618,7 @@ void TorController::disconnected_cb(TorControlConnection& conn)
     if (!reconnect)
         return;
 
-    LogPrint("tor", "tor: Disconnected from Tor control port %s, trying to reconnect\n", target);
+    LogPrint("tor", "tor: Not connected to Tor control port %s, trying to reconnect\n", target);
```
Comment (laanwj) [inline src/torcontrol.cpp]: I'm confused. Didn't this get in in a previous change?

Comment (MarcoFalke) [inline src/torcontrol.cpp]: https://github.com/bitcoin/bitcoin/pull/7035#issuecomment-160808428

Comment (laanwj) [inline src/torcontrol.cpp]: Oh I thought he fixed that...

**Comment by MarcoFalke** [comment]
Ok then, let's not do a backport.

**Pull request merged**